### PR TITLE
Resolving import on load of `config.json` and `env.json`

### PR DESCRIPTION
**Pull request by pvenkatakrishnan**

### CODE REVIEW

**Comment by aredridel** [comment]
:+1:

**Comment by pvenkatakrishnan** [comment]
@jasisk review ?

File: lib/factory.js [hunk 1]
```diff
@@ -56,7 +58,6 @@ export default class Factory {
 
     create(cb) {
         this.promise
-            .then(store => Handlers.resolveImport(store, this.basedir))
```
Comment (jasisk) [inline lib/factory.js:59]: Wondering if there is ever a time that this would still be handy?
`npm start --services='import:overrides.json'`

**Comment by jasisk** [comment]
I'm :+1: besides my outstanding Q.

**Comment by pvenkatakrishnan** [comment]
thats a very good point... yes since we have changed the order in which merging is done, this will fail without import resolution on create... will add that back in.

**Pull request merged**